src/js/App/ErrorComponent/ErrorComponent.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import * as Sentry from '@sentry/browser';
+import {
+  Bullseye,
+  Button,
+  EmptyState,
+  EmptyStatePrimary,
+  EmptyStateBody,
+  EmptyStateIcon,
+  ExpandableSection,
+  Title,
+  Text,
+  TextContent,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import ErrorCircleOIcon from '@patternfly/react-icons/dist/js/icons/error-circle-o-icon';
+import { getUrl } from '../../utils';
+
+import './ErrorComponent.scss';
+
+const ErrorComponent = (props) => {
+  useEffect(() => {
+    Sentry.captureException(new Error('Unhandled UI runtime error'), {
```

Comment:
@ryelo is correct? Do we have different sentry instances for different apps? ideally, it should log the error for the specific application.

Reply:
The sentry logs it based on the bundle etc, AFAIK we don't have to check which app is currently active.

Reply:
You can do this above and you're fine. You don't have to check anything in particular.